### PR TITLE
TRUNK 5441, 5442, 5443: Move comments, reduce complexity, rename logger

### PR DESCRIPTION
**Pull request by siddhpant**
Fixed some open issues in Sonar.

https://issues.openmrs.org/browse/TRUNK-5441
https://issues.openmrs.org/browse/TRUNK-5442
https://issues.openmrs.org/browse/TRUNK-5443

### CODE REVIEW

**Comment by samuelmale** [comment]
@siddhpant  did you get a chance and read the following resources?
https://wiki.openmrs.org/display/docs/Pull+Request+Tips,

* Can you improve your commit message and PR title 
* You should also squash your commits into one
* You should also share links to the Jira issue and your GCI task instance

**Comment by siddhpant** [comment]
Links to the issues can be found in commit messages. These are SonarQube issues and not Jira issues.

Regarding squashing, I already committed and then saw another issue. :(

I think the commit title and message are to the point? So it would be redundant to type the same thing in the PR message....

**Comment by samuelmale** [comment]
@siddhpant did you read this https://wiki.openmrs.org/display/docs/Pull+Request+Tips?

**Comment by siddhpant** [comment]
I did.

These are just codestyle fixes, no major issues. Also, I can't login to Jira (I don't have permission). 

Besides, I don't think I should make Jira issue for moving comment to new line or making an if block which was essentially already present.

**Comment by samuelmale** [comment]
If you find it too trivial or hard to follow existing conventions, you can always get back to the community through Openmrs talk and table your ideas. Otherwise I think it's of help to me and you to learn and use these conventions.

If you can't login on Jira, that's a known issue. Just make a query at the help desk asking for permission. Btw can you post your blocker on Talk?

**Comment by siddhpant** [comment]
Applied for access.

**Comment by dkayiwa** [comment]
@siddhpant did you see the merge conflicts here?

**Comment by siddhpant** [comment]
Added jira links

**Comment by dkayiwa** [comment]
Can you squash these commits into one?
Also include the ticket id in the commit message as advised at https://wiki.openmrs.org/display/docs/Pull+Request+Tips
